spec.html

```diff
@@ -152,7 +152,7 @@ <h1>Properties of the _AggregateError_ Prototype Object</h1>
       <ul>
         <li>is an ordinary object.</li>
         <li>is not an Error instance and does not have an [[ErrorData]] internal slot.</li>
-        <li>has a [[Prototype]] internal slot whose value is the intrinsic object %Error.prototype%.</li>
+        <li>has a [[Prototype]] internal slot whose value is the intrinsic object %ErrorPrototype%.</li>
```

Comment:
Why was this reverted to the old intrinsic style?

Reply:
Oops I was looking at the old version of ecma262 repo and copied this from "Properties of the NativeError Prototype Objects"